roles/confluent.test.ldap/tasks/tls_custom_certs.yml

```diff
@@ -18,3 +18,4 @@
     src: "{{ssl_key_filepath}}"
     dest: "/var/ssl/private/ldaps/{{ssl_key_filepath|basename}}"
     mode: 0666
+  diff: "{{ not mask_sensitive_diff|bool }}"
```

Comment:
the test roles are fine to leave as is, those arent meant for customer use

Reply:
Done